commit ffd3e38977685ae119bd00d05383c81a01ad0c48
Author: Marina Glancy <marina@moodle.com>
Date:   Thu Jul 7 15:23:29 2011 +0800

    MDL-28197 adding slashes to group name when creating group in user upload

diff --git a/admin/uploaduser.php b/admin/uploaduser.php
index 52834a5..f74967c 100755
--- a/admin/uploaduser.php
+++ b/admin/uploaduser.php
@@ -665,7 +665,7 @@ if ($formdata = $mform->is_cancelled()) {
                     $newgroupdata = new object();
                     $newgroupdata->name = $addgroup;
                     $newgroupdata->courseid = $ccache[$shortname]->id;
-                    if ($ccache[$shortname]->groups[$addgroup]->id = groups_create_group($newgroupdata)){
+                    if ($ccache[$shortname]->groups[$addgroup]->id = groups_create_group(addslashes_recursive($newgroupdata))){
                         $ccache[$shortname]->groups[$addgroup]->name = $newgroupdata->name;
                     } else {
                         $upt->track('enrolments', get_string('unknowngroup', 'error', $addgroup), 'error');

commit 682e430ce090058889f219607711e09c634e9e90
Author: Marina Glancy <marina@moodle.com>
Date:   Fri Jul 8 10:10:41 2011 +0800

    MDL-28197 another possible sql vulnerability injection fixed

diff --git a/admin/uploaduser.php b/admin/uploaduser.php
index f74967c..f917566 100755
--- a/admin/uploaduser.php
+++ b/admin/uploaduser.php
@@ -558,7 +558,7 @@ if ($formdata = $mform->is_cancelled()) {
                 continue;
             }
             // save custom profile fields data
-            profile_save_data($user);
+            profile_save_data(addslashes_recursive($user));
 
             // make sure user context exists
             get_context_instance(CONTEXT_USER, $user->id);
